AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -28,6 +28,10 @@ instagram.com##._aagw:empty
 !
 ! End: Instagram.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176470
+venturebeat.com##.post-boilerplate:has(.ICCta)
+venturebeat.com##.sidebar-widget-featured-event
```

Comment:
We already have this rule
![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/48119277-7d48-400f-9812-4570e3645ec4)
```suggestion
```